### ISSUE TITLE
fix issue causing CI to fail 

### ISSUE THREAD

**Issue by tgrecojs** [lari-labs/dapp-ertp-airdrop#57]

**Comment by dckc** [comment]
I hope this issue gets priority soon.

In particular, I hope branch protection is turned on so that code can't land until tests and lint pass.

**Comment by tgrecojs** [comment]
all signs point to this stemming from noble-hashes.... here is the stack trace I managed to produce:

```
not ok 7 - Error: Cannot find module '/Users/tgreco/dapp-airdrop/contract/node_modules/@noble/hashes/pbkdf2.js'
  ---
    code: MODULE_NOT_FOUND
    path: /Users/tgreco/dapp-airdrop/contract/node_modules/@noble/hashes/package.json
    name: Error
    message: >-
      Cannot find module
      '/Users/tgreco/dapp-airdrop/contract/node_modules/@noble/hashes/pbkdf2.js'
    at: ''
  ...
```

It's that pbkdf2.js file specifically, so I just took quite a bit of time attempting to fix.

I tried:
 - building @noble/hashes inside cosmjs/crypto
 - replacing the @noble/hashes with the one that exists in dapp-ertp/airdrop/contract/vendor
 - patching cosmjs/crypto to reference the @vendor/@noble/hashes
 - building cosmjs' packages in a seperate directory -> copying the necessary directories into contract/node_modules/@cosmjs

I'm confused as to why it's falling now seeing as it's relied on this library from the jump (or at least since the time this code was written). Regardless of my confusion, the issue still exists.

If you have any ideas as to how this could be fix please let me know.

**Comment by dckc** [comment]
What I'm used to is that the CI build represents whatever is currently working. Right now, all CI builds are failing, so it looks like _nothing_ is working. If a test is failing in CI and it's not pretty obvious how to fix it, then that part of the project is not working; it deserves its own issue:

 1. Open an issue to represent the thing that's failing in CI (include pointers to CI logs)
 2. Turn off the test, adding a pointer to the new issue.

In [a recent build log](https://github.com/hindley-milner-systems/dapp-ertp-airdrop/actions/runs/10782528747/job/29902678171), I see those diagnostics under `ertp-airdrop-ui` / `vitest spec`.

So consider opening an issue such as "@noble/hashes/pbkdf2 not found in ui" and  in `ui/package.json`, changing

```json
    "test": "vitest spec",
```

to

```json
    "test": "echo UI testing postponed pending issue 1234; exit 0",
    "test-pending-1234": "vitest spec",
```

cc @Jovonni

**Comment by tgrecojs** [comment]
this is fixed in #105  

https://github.com/hindley-milner-systems/dapp-ertp-airdrop/actions/runs/10800676702
